products/virtual-agent/research/inclusive-design-interviews/inclusive-design-interviews-conversation-guide.md

```diff
@@ -18,39 +18,139 @@
 
+## Tech set-up
+
+Next, I want to make sure your technology is ready to go for this session. Shannon may have gone over a list of technical questions when you registered.
+- I'm going to quickly run through those again just to make sure I have the right information.
+
+Checklist:
+- Are you using a computer or mobile phone during our session today? 
+- Mac or Windows? 
+- iOS or Android?
+   - If not joined by Zoom already:
+   - Do you have Zoom installed? 
+- Are you wearing headphones today? If so, would it be possible for you to take them off and use your computer or mobile audio? I'm asking so that I can hear your screen reader during the session.
+- What browser are you using today?
+- Do you have a screen reader on this device?
+- What kind of screen reader do you use?
+- How long have you been using it?
+- Are you using a screen magnifier today?
+- Are there any other assistive technologies that you use?
+  - [if low vision, the participant may require captions which need to be turned on in advanced settings]
+- Do you know your screen reader's speaking rate (verbosity setting)? High, medium, low? 
+  - [If high, be prepared to have them repeat anything that can't be heard or understood]
+   - I may need to pause you at times an ask you what you're hearing if I can't make out what's being read.
+
+**For those joined by iPhone and Zoom already**
+SKIP TO [Screen sharing](#screen-sharing) SET UP
+
+## Joining Zoom
+
+### Help participant join with device
+
+- I'm going to send you a zoom link [by format specified] for you to open on your phone so you can join this meeting.  
+- [Send through Veteran's preferred communication channel]
+- When you open the link, you should be taken to the Zoom application automatically. Zoom will ask for access to your microphone and video. We only need your audio for this session, so please tap "Allow" for the microphone prompt. Feel free to tap "Don't Allow" for video permissions if you are not comfortable having your face on camera.
+- Upon joining the room, Zoom will have you turn on audio by asking “To hear others, please join audio.” Please swipe to and double tap "Wifi or Cellular Data." [This button will be at the center of their screen.]
+- Once you're in the call...
+
+## Screen sharing
+
+At this point [I'd like you to go back to Zoom] I'm going to have you share your Zoom screen so that I can follow along with you. 
+
+### VoiceOver iOS (Mobile and Tablet) using web app
+
+ - Have you shared your screen before on Zoom?
+[If no] Please open your iPhone settings. Once you’re there, go to your control center. Then, find the “Insert screen recording” button and double tap to activate it. When you’re done, please return back to the zoom app.
+ - Within Zoom, I'd like you to find the "Share content" button and double tap it. You may have to swipe a number of times.
```

Comment:
What do you think about rewriting **Within Zoom, I'd like you to find the "Share content" button and double tap it.** to say **Within Zoom, select the "Share content" button and double tap it.**

Reply:
@1Copenut I think that works. Thanks for the suggestion.